### ISSUE TITLE
CivetQC: refuse to work when any of the CivetOutput is archived.

### ISSUE THREAD

**Issue by prioux** [aces/cbrain-plugins-neuro#111]
The CivetQC task should refuse to run if any of the CivetOutputs is archived; they check could be made both on the portal side when launching the task, and on the Bourreau side (if the task is being restarted after some time for instance, and some of the CivetOutputs have been archived in the meantime).

**Comment by crocodoyle** [comment]
Maybe the CivetQC task could automatically dearchive CivetOutputs instead of failing the task?

**Comment by prioux** [comment]
Yes, it's a possibility, but it could be enormously costly. Assume a study with 100 CivetOutputs located on a remote DataProvider, then for each study a call to `object.provider_unarchive` implies:

* syncing from the remote DataProvider the archived CivetOutput to the local cache
* unarchiving it in the cache
* sending back the unarchived CivetOutput to the DataProvider

All of this would have to be performed during the `Setup` state of the task, since it's the only place where we can execute Ruby code, and this would block one Bourreau subworker for the entire duration of the state.

I'm not saying it's impossible, just maybe it's better to tell users to unarchive their CivetOutputs first separately, because this can be done without blocking anything else.

**Comment by crocodoyle** [comment]
OK I think I understand the problem with my suggestion. When a user unarchives a Userfile, it uses the background processing of the BrainPortal, and when a Task unarchives a file, it has to query the BrainPortal to do it, which blocks the Bourreau, which is waiting for the Userfiles to be unarchived....

Maybe in the Task setup (on the BrainPortal side), there could be an "unarchive all" button that would launch unarchives for all the files? This could be standard across all Tools, because I don't think any Tools support archived inputs, do they?
